How-To-Release-Riak.md

```diff
@@ -0,0 +1,56 @@
+This doc contains the internal Basho policy for building, testing, and releasing Riak.
+
+# Scheduling
+All public releases must conform to the following schedule:
+* An expected delivery date on a Wednesday. Each delivery date must have at least one fallback date that is no earlier than one week after the expected delivery date.
+* A Go/No-Go scheduled on the Thursday after the expected delivery date or fallback date to ensure that the public release can proceed as planned. During the Go/No-Go we must ensure:
+  * Giddyup has been triaged, and there are no major issues
+  * Smoke testing has completed successfully on all supported platforms
+  * Documentation content has been merged and triaged if necessary, and release notes are ready to ship
+  * Packages for all supported platforms have been built and are ready to stage
+  * Any marketing announcements have been reviewed and are ready to ship
+* A final staging date on the Friday after the Go/No-Go. During this time, all packages will be pushed to their final download locations, the `External Release` docs will be sent around internall for review, and the indexes for the download pages will be generated
```

Comment:
something like this: https://github.com/basho/internal_wiki/wiki/riak-2.0.8-External-Release

I don't know really what to call it, since it's for internal use only, but contains the links for the external packages. I'm open to suggestions.